sdk/identity/identity/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@azure/identity",
   "sdk-type": "client",
-  "version": "2.1.1",
+  "version": "3.0.0-beta.1",
```

Comment:
What motivates the change to 3.0.0-beta.1?

Reply:
This is the plan for copper. These are supposed to be part of the new GA along with other features we'll be adding